ntbs-service/Pages/Notifications/Edit/Children/ManualTestResult.cshtml

```diff
@@ -13,121 +13,127 @@
         <h3> @(Model.RowId == null ? "Add new " : "Edit ") test result</h3>
         <form method="post" autocomplete="off">
             <nhs-grid-row>
-                <nhs-grid-column grid-column-width="OneHalf">
+                <nhs-grid-column grid-column-width="Full">
```

Comment:
Semantically slightly iffy use of columns, but I think it has a better result on render - don't know if it was mandated with Nancy to push these onto 2 lines - I just don't think the date plays well with other inputs on the same line.
I tried thirds for the other inputs, and that looked a bit iffy too.

Reply:
I think the semantics are actually fine, in fact, i would consider using a single row with all columns as you have them.
as for the changed layout - do verify with Nancy, but LGTM!